pages/agent/v3/help/_bootstrap.md

```diff
@@ -18,7 +18,7 @@ script.
 
 ## Description
 
-The bootstrap command executes a buildkite job locally.
+The bootstrap command executes a Buildkite job locally.
```

Comment:
I realize this is in a generated file. I'm going to open a PR against buildkite/agent, but I wanted to make the build green first.

Reply:
Please take files from the `help` out of this PR. This is some delicate ground we're treading, it can break more than just docs.